doc/source/ray-air/examples/analyze_tuning_results.ipynb

```diff
@@ -0,0 +1,918 @@
+    "    \"tree_method\": \"approx\",\n",
+    "    \"objective\": \"multi:softmax\",\n",
+    "    \"eval_metric\": [\"mlogloss\", \"merror\"],\n",
+    "    \"num_class\": 8,\n",
+    "}\n",
+    "\n",
+    "trainer = XGBoostTrainer(\n",
+    "    scaling_config={\"num_workers\": 2},\n",
+    "    label_column=\"target\",\n",
+    "    params=params,\n",
+    "    datasets={\"train\": train_dataset},\n",
+    "    num_boost_round=10,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e436035b",
+   "metadata": {},
+   "source": [
+    "The trainer contains a few default parameters. We can now create the Tuner with a search space for other parameters. During the run, it will then overwrite the default parameters in the trainer according to the search specification.\n",
+    "\n",
+    "Here, we just want to tune the XGBoost `max_depth` parameter. Additionally, we want to compare our two `preprocessor` pipelines. Lastly, we configure Tune to minimize the `train-mlogloss` metric. In random search, this doesn't affect the evaluated configurations, but it will affect our default results fetching for analysis later."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "c5d00628",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tuner = Tuner(\n",
+    "    trainer,\n",
+    "    param_space={\n",
+    "        \"params\": {\"max_depth\": tune.randint(2, 8)},\n",
+    "        \"preprocessor\": tune.grid_search([prep_v1, prep_v2]),\n",
+    "    },\n",
+    "    tune_config=TuneConfig(num_samples=4, metric=\"train-mlogloss\", mode=\"min\"),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "378128cc",
+   "metadata": {},
+   "source": [
+    "Let's run the tuning. This will take a few minutes to complete."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "ab642705",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "== Status ==<br>Current time: 2022-05-09 17:29:41 (running for 00:04:05.52)<br>Memory usage on this node: 9.3/16.0 GiB<br>Using FIFO scheduling algorithm.<br>Resources requested: 0/16 CPUs, 0/0 GPUs, 0.0/4.63 GiB heap, 0.0/2.0 GiB objects<br>Current best trial: 4544b_00000 with train-mlogloss=0.559286 and parameters={'params': {'max_depth': 7}, 'preprocessor': StandardScaler(columns=['Wilderness_Area_0', 'Wilderness_Area_1', 'Wilderness_Area_2', 'Wilderness_Area_3'], ddof=0, stats={'mean(Wilderness_Area_0)': 0.4488650836815763, 'mean(Wilderness_Area_1)': 0.051434393781884025, 'mean(Wilderness_Area_2)': 0.4360736094951567, 'mean(Wilderness_Area_3)': 0.06362691304138296, 'std(Wilderness_Area_0)': 0.4973783472700716, 'std(Wilderness_Area_1)': 0.22088208826922595, 'std(Wilderness_Area_2)': 0.4958965785292558, 'std(Wilderness_Area_3)': 0.2440871339874498})}<br>Result logdir: /Users/kai/ray_results/XGBoostTrainer_2022-05-09_17-25-34<br>Number of trials: 8/8 (8 TERMINATED)<br><table>\n",
```

Comment:
Let's decrease the verbosity for the example? I think @Yard1 is enabling this by default right?

Reply:
Verbosity will be set to 1 by default (simple trial results)